dataland-qa-service/src/main/kotlin/org/dataland/datalandqaservice/entities/ReviewInformationEntity.kt

```diff
@@ -16,4 +17,20 @@ data class ReviewInformationEntity(
     val receptionTime: Long,
     var qaStatus: QaStatus,
     val reviewerKeycloakId: String,
-)
+    var message: String?,
+) {
+    /**
+     * Converts the ReviewInformationEntity into a ReviewInformationResponse that is used in a response for a
+     * GET Request.
+     * The ReviewInformationResponse can optionally hide the reviewerKeycloakId and this function respects this.
+     */
+    fun toReviewInformationResponse(shouldHideReviewKeycloakId: Boolean): ReviewInformationResponse {
+        return ReviewInformationResponse(
+            dataId,
+            receptionTime,
+            qaStatus,
+            if (shouldHideReviewKeycloakId) null else reviewerKeycloakId,
```

Comment:
Was passiert wenn der Boolean "shouldHideReviewKeycloakId" null ist? Dann würde man die reviewerKeycloakId sehen oder?
Das sollten wir vermeiden. Vll sollten wir die Abfrage umdrehen und in die Richtung "showReviewKeycloakId" gehen. Was meint ihr?

Reply:
Ich habe die Variable umbenannt. Das sollte intuitiver sein und damit fällt auch die Negierung weg. 
Andererseits ist mein Verständnis von dem Booleantyp in Kotlin so, dass dieser nicht null sein kann (im Vgl. zu Boolean?) und daher, die alte Implementierung genauso funktionieren sollte.

Reply:
Ja fair. Ich habe übersehen dass der Input als nicht nullable definiert wurde, damit hast du in der Tat recht, dass der Wert null niemals annehmen sollte.